packages/block-editor/src/components/block-edit/index.js

```diff
@@ -71,7 +70,6 @@ export default function BlockEdit( {
 					[ mayDisplayParentControlsKey ]: mayDisplayParentControls,
 					[ blockEditingModeKey ]: blockEditingMode,
 					[ blockBindingsKey ]: bindings,
-					[ isPreviewModeKey ]: isPreviewMode,
```

Comment:
I'm not sure we want to allow access to this value from the edit context. Alternatively, I think we also have a "private context" that we can move this config into. Maybe @ellatrix has an opinion here.

Reply:
I see. From what I can see the private context is for the block list component, not block edit. I'll revert the change and continue using the Symbol here.